corgi/tasks/tagging.py

```diff
@@ -0,0 +1,58 @@
+from celery.utils.log import get_task_logger
+from celery_singleton import Singleton
+
+from config.celery import app
+from corgi.core.models import Product, ProductStream, ProductStreamTag, ProductVersion
+from corgi.tasks.common import RETRY_KWARGS, RETRYABLE_ERRORS
+
+logger = get_task_logger(__name__)
+
+
+@app.task(
+    base=Singleton,
+    autoretry_for=RETRYABLE_ERRORS,
+    retry_kwargs=RETRY_KWARGS,
+)
+def apply_tags():
+    apply_stream_no_manifest_tags()
+
+
+def apply_stream_no_manifest_tags():
+    tag_name = "no_manifest"
+    tag_value = True
+    apply_middleware_stream_no_manifest_tags(tag_name, tag_value)
+    apply_rhel_8_9_z_stream_no_manifest_tags(tag_name, tag_value)
+    apply_managed_services_no_manifest_tags(tag_name, tag_value)
```

Comment:
I guess we need to always pass a tag_value so that get_or_create uses an index and is atomic. But let's use an empty "" string default instead of a literal True / bool value.
```suggestion
    # Must pass tag_value to get_or_create so it's atomic
    tag_value = ""
    apply_middleware_stream_no_manifest_tags(tag_name, tag_value)
    apply_rhel_8_9_z_stream_no_manifest_tags(tag_name, tag_value)
    apply_managed_services_no_manifest_tags(tag_name, tag_value)
```

Reply:
OK,  updated to empty string.